pages/[category]/index.tsx

```diff
@@ -1,51 +1,155 @@
+  if (!validCategory) return <Error statusCode={404} />
   return (
     <>
       <Head>
         <title>{`${siteTitle} | ${displayCategory}`}</title>
       </Head>
-      <RecordPane
-        orgInfo={fetchedRecords}
-        displayCategory={displayCategory}
-        routeCategory={routeCategory}
-        setRecords={setFetchedRecords}
-      />
-
-      {convertedLocRecords && <DisplayMap latLongInfo={convertedLocRecords} />}
+      <div className={classes.landingPageContainer}>
+        <Grid container>
+          <DisplayCategoryImage
+            displayCategory={displayCategory}
+            routeCategory={routeCategory}
+          />
+          <Hidden smDown>
+            <CategoryDescription
+              displayDescription={displayDescription}
+              activeCopy={activeCopy}
+            />
+          </Hidden>
+          <Hidden smDown>
+            <DesktopFilterView
+              fields={fields}
+              handleFieldsSelected={handleFieldsSelected}
+              routeCategory={routeCategory}
+              activeCopy={activeCopy}
+            />
+          </Hidden>
+          <Hidden mdUp>
+            <MobileButtonsLandingPage
+              activeCopy={activeCopy}
+              setOpen={setOpen}
+            />
+          </Hidden>
+          <Modal
+            open={open}
+            onClose={() => setOpen(false)}
+            aria-labelledby="simple-modal-title"
+            aria-describedby="simple-modal-description"
+          >
+            <>
+              <MobileFilterModal
+                fields={fields}
+                handleFieldsSelected={handleFieldsSelected}
+                routeCategory={routeCategory}
+                setOpen={setOpen}
+                activeCopy={activeCopy}
+              />
+            </>
+          </Modal>
+          <Hidden mdUp>
+            <div
+              style={{
+                display: isListView && !isMapView ? 'contents' : 'none',
+              }}
+            >
+              <RecordPane
+                orgInfo={filteredResults}
+                displayCategory={displayCategory}
+                routeCategory={routeCategory}
+                setRecords={setFetchedRecords}
+              />
+            </div>
+            {convertedLocRecords && (
+              <div
+                style={{
+                  display: isMapView && !isListView ? 'contents' : 'none',
+                }}
+              >
+                <DisplayMap latLongInfo={convertedLocRecords} />
+              </div>
+            )}
+          </Hidden>
+          <Hidden smDown>
+            <RecordPane
+              orgInfo={filteredResults}
+              displayCategory={displayCategory}
+              routeCategory={routeCategory}
+              setRecords={setFetchedRecords}
+            />
+            {convertedLocRecords && (
+              <DisplayMap latLongInfo={convertedLocRecords} />
+            )}
+          </Hidden>
```

Comment:
I think the issue is still with these and with the hidden components. I can see from the network tab it is still reloading the map when the screen is resized. From what I can tell, the way the hidden component works is that it loads content to the dom when the a breakpoint condition is met. As these conditions are met, it is loading or unloading the components within. So, I think that we are going to continue to have the double map loading problem for as long as there are two components.